spec/markdown/rest-api.md

```diff
@@ -0,0 +1,111 @@
+## REST API
+
+The following sections define the Sidetree resolution and operations endpoints. Please refer to the companion [Sidetree REST API](https://identity.foundation/sidetree/docs/swagger/) specification for additional information, as well as REST API definitions for blockchain and CAS components.
+
+### Sidetree Resolution
+
+### Sidetree Operations
+
+Sidetree operation requests to the REST API consist of a type property indicating the operation to be performed along with operation-specific properties and data.
+
+::: example
+```json
+{
+    "type": OPERATION_TYPE,
+    ...
+}
+```
+:::
+
+A valid Sidetree Operation Request is a JSON document composed as follows:
+
+1. The Operation Request MUST contain a `type` property, and its value MUST be a valid operation defined in
+[File Structure](#file-structures). The defined operations are `create`, `recover`, `deactivate`, `update`.
+2. Populate additional properties according to the appropriate subsection.
+3. The client MUST POST the Operation Request JSON document to the Sidetree operation endpoint `/sidetree/operations` under the desired REST server path.
+4. The server MUST respond with HTTP status 200 when successful. Otherwise, for failure, the server MUST respond with an appropriate HTTP Status Code (400, 401, 404, 500).
+
+#### Create
```

Comment:
I would prefer to see real JWS here... like https://identity.foundation/.well-known/resources/did-configuration/#json-web-token-proof-format